web/beacon-app/src/features/auth/types/RegisterService.ts

```diff
@@ -20,16 +20,16 @@ export interface NewUserResponseData {
   created: string;
 }
 
-export type NewUserAccount = Omit<User, 'user_id'>;
+export type NewUserAccount = Omit<
+  User,
+  'user_id' | 'name' | 'organization' | 'domain' | 'terms_agreement' | 'privacy_agreement'
+>;
 
 export type NewInvitedUserAccount = Omit<User, 'user_id' | 'organization' | 'domain'>;
 
-export const hasUserRequiredFields = (
-  account: NewUserAccount
-): account is Required<NewUserAccount> => {
-  return (
-    Object.values(account).every((x) => !!x) &&
-    account.terms_agreement === true &&
-    account.privacy_agreement === true
-  );
+export const hasUserRequiredFields = (account: NewUserAccount | NewInvitedUserAccount): boolean => {
+  if (!account.email || !account.password || !account.pwcheck) {
+    return false;
+  }
+  return true;
```

Comment:
```suggestion
 Object.values(account).every((x) => !!x) // is this not working ?
```

Reply:
I think I got a failing test when I tried that. I'll try again before merging because I ended up trying a few different things yesterday, so I may have gotten things a little confused at some point.